WordPressVIPMinimum/Sniffs/Security/ProperEscapingFunctionSniff.php

```diff
@@ -24,9 +24,15 @@ class ProperEscapingFunctionSniff extends Sniff {
 	 * @var array
 	 */
 	public $escaping_functions = [
-		'esc_url',
-		'esc_attr',
-		'esc_html',
+		'esc_url'    => 'url',
+		'esc_attr'   => 'attr',
+		'esc_attr__' => 'attr',
+		'esc_attr_x' => 'attr',
+		'esc_attr_e' => 'attr',
+		'esc_html'   => 'html',
+		'esc_html__' => 'html',
+		'esc_html_x' => 'html',
+		'esc_html_e' => 'html',
 	];
```

Comment:
Have you considered code like this ?
```php
<div><?= esc_attr($attr) ?></div><!-- Error -->
<div><?php esc_attr_e($attr) ?></div><!-- Error -->
```

(Sniff failure on this code is only partially due to the additional functions being added)